sites/public/src/forms/filters/FilterForm.tsx

```diff
@@ -300,8 +302,8 @@ const FilterForm = (props: FilterFormProps) => {
           />
         </GridCell>
       </GridSection>
-      <div className="text-left mt-8 bg-white border-t border-gray-450 pt-4">
-        <div className={"pb-4 mb-2 px-4"}>
+      <div className="text-left mt-28 bg-white border-t border-gray-450 pt-4">
```

Comment:
<img width="335" alt="Screen Shot 2022-04-25 at 10 53 08 PM" src="https://user-images.githubusercontent.com/65367387/165224688-20697189-a0ec-40e7-a6e1-2bd1ad5191e5.png">
A static margin like this will make the footer push to the bottom properly on a desktop view, but not elsewhere, where it will create an unnecessary gap.